internal/telemetry/cluster_test.go

```diff
@@ -40,6 +40,32 @@ func TestNodeCountInAClusterWithOneNode(t *testing.T) {
 	}
 }
 
+func TestClusterIDRetrievesK8sClusterUID(t *testing.T) {
+	t.Parallel()
+
+	c := newTestCollectorForCluserWithNodes(t, node1, kubeNS)
+
+	got, err := c.ClusterID(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	want := "329766ff-5d78-4c9e-8736-7faad1f2e937"
```

Comment:
where do this value come from, will `c.ClusterID(context.Background()` always return this value?

Reply:
it's coming from fake testClient configured with `cube-system` (var `kubeNS`) Obj.